homeassistant/components/brother/sensor.py

```diff
@@ -20,9 +23,11 @@
 ATTR_REMAINING_PAGES = "remaining_pages"
 
 
-async def async_setup_entry(hass, config_entry, async_add_entities):
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: Callable
```

Comment:
nit: `AddEntitiesCallback` from https://github.com/home-assistant/core/pull/49812 can now be used here